src/pkg/packager/sources/split.go

```diff
@@ -85,6 +85,12 @@ func (s *SplitTarballSource) Collect(dir string) (string, error) {
 		if _, err = io.Copy(pkgFile, f); err != nil {
 			return "", fmt.Errorf("unable to copy file %s: %w", file, err)
 		}
+
+		// Close the file when done copying
+		err = f.Close()
+		if err != nil {
+			return "", fmt.Errorf("unable to close file %s: %w", file, err)
+		}
```

Comment:
```suggestion
		if err := f.Close(); err != nil {
			return "", fmt.Errorf("unable to close file %s: %w", file, err)
		}
```
Back to style conversation here https://github.com/defenseunicorns/zarf/issues/2312, do we want to enforce things like this. Something to discuss